src/hooks/useNFTs.ts

```diff
@@ -10,6 +10,38 @@ import { GlobalNom, NomResult } from "./useNom";
 import { SUPPORTED_NETWORKS } from "config";
 import { JsonRpcProvider } from "@ethersproject/providers";
 
+async function poapCall(resolution:string) {
+
+  if (process.env.REACT_APP_POAP_KEY === undefined){
+    console.error("REACT_APP_POAP_KEY not present, skipping POAP fetch");
+    return [];
+  }
+  const poapMetadata = [];
+  const options = { //POAP API parameters
+    method: 'GET',
+    headers: {
+      Accept: 'application/json',
+      'X-API-Key': process.env.REACT_APP_POAP_KEY,
+    }
+  };
+  
+  var poaps:any; 
```

Comment:
use `let`, not `var`

https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var